### ISSUE TITLE
[Bug][a11y] Cannot type the email address when sharing a page in gmail

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#3576]
## Steps to reproduce
1. Turn Talkback or Voice Assistant on.
1. Open a page and share it through Gmail.
1. Type the email address.

### Expected behavior
The letters from the email address should be displayed.

### Actual behavior
The Voice assistant recognizes the letters, reads them but nothing is displayed.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8)
* Fenix version: 1.0.0-rc.2 (#11700147)

![share](https://user-images.githubusercontent.com/40694158/59839567-796bef80-9359-11e9-84b8-ad585e1dc63b.gif)

**Comment by softvision-miralobontiu** [comment]
duplicated the issue :(
